Game.java

```diff
@@ -34,18 +35,14 @@ public String showBoard(Board board) {
     }
 
     public Board playerMakesMove() {
-        int index = randInt(0, board.availablePositions().size());
-        int pos;
-        if (turn == 'o' && gameMode.equals("pvc")) {
-            pos = (int) board.availablePositions().get(index);
-            board = board.playerMakesMove('o', pos);
+        if (getTurn() == 'o' && gameMode.equals("pvc")) {
+            board = board.playerMakesMove(getTurn(), computerPlayer.nextMove(board));
         } else if(gameMode.equals("cvc")) {
-            pos = (int) board.availablePositions().get(index);
-            board = board.playerMakesMove(getTurn(), pos);
+            board = board.playerMakesMove(getTurn(), computerPlayer.nextMove(board));
         } else {
-            pos = scanner.nextInt();
-            if (!board.availablePosition(pos - 1)) return board;
-            board = board.playerMakesMove(getTurn(), pos - 1);
+            int position = scanner.nextInt();
```

Comment:
this condition's get next move functionality could be transferred to a ConsolePlayer or HumanPlayer or some other creation!

Then you might see some duplication that can be refactored.

There are a few steps involved here so this may be worth creating a story for.  We can discuss more in the morning.